front/src/usermanagement/UserManagementView.js

```diff
@@ -289,6 +317,29 @@ class UserManagementView extends Component {
       this.handleChangePassClose();
     }
   }
+  // ___________________________________________________________________
+  // Closes dialog for adding email for some1 else
+  handleaddEmailClose(e) { // eslint-disable-line
+    this.setState({
+      email: '',
+      addEmailDialogOpen: false,
+    });
+  }
+
+  // Adds email for some1 else by their ID
+  async handleaddEmailCloseConfirm() {
+    const response = await addEmail(this.findUserId(), this.state.email);
+    if (!response) {
+      this.setState({
+        mokatVaihdossa: true,
+
+      });
+      console.log('OH NO');
+    } else {
```

Comment:
OH NO! (change or delete this consolelog)

Reply:
done